src/components/report/Summary.js

```diff
@@ -9,7 +9,32 @@ export default {
 	},
 	computed: {
 		featureLength() {
-			return Object.keys(this.features).length;
+			const count = Object.keys(this.features).length;
+			return count || "no";
+		},
+		axesCount() {
+			let count = 0;
+			if (this.font.variable) {
+				count = this.font.variable.axes.length;
+			}
+			return count || "no";
+		},
+		instancesCount() {
```

Comment:
Previous comment also applies to this function.

Reply:
It is possible for a variable font to have no instances. This code returns `no` for that situation, _and_ if the font isn;t variable to begin with.